packages/app/src/systems/Core/components/MigrationWarning.tsx

```diff
@@ -0,0 +1,38 @@
+import { Alert } from "@fuel-ui/react";
+import { useState } from "react";
+
+const LOCALSTORAGE_MIGRATION_WARNING = "fuel--migration-warning";
+
+const useMigrationWarning = () => {
+  const [hide, setToHide] = useState(
```

Comment:
can we use `atom` for it?

```
import { atom, useAtom } from "jotai";

const migrationHidden = atom(false);

const useMigrationWarning = () => {
  const [isHidden, setIsHidden] = useAtom(migrationHidden);
}
```

Reply:
we need to persist in order to not show again